src/tree/SubscriptionTreeItem.ts

```diff
@@ -72,13 +75,21 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         wizardContext.repoHtmlUrl = await tryGetRemote(wizardContext);
         wizardContext.telemetry.properties.gotRemote = String(!!wizardContext.repoHtmlUrl);
 
+        wizardContext.fsPath = getSingleRootFsPath();
+
         await wizard.prompt();
         const newStaticWebAppName: string = nonNullProp(wizardContext, 'newStaticWebAppName');
         wizardContext.newResourceGroupName = newStaticWebAppName;
 
         await wizard.execute();
         context.showCreatingTreeItem(newStaticWebAppName);
 
+        if (wizardContext.fsPath) {
+            await updateWorkspaceSetting(appSubpathSetting, wizardContext.appLocation, wizardContext.fsPath);
+            await updateWorkspaceSetting(apiSubpathSetting, wizardContext.apiLocation, wizardContext.fsPath);
+            await updateWorkspaceSetting(appArtifactSubpathSetting, wizardContext.appArtifactLocation, wizardContext.fsPath);
+        }
```

Comment:
> But I figured when we have the concept of a basic create, we can auto-populate the value with these settings.

You know you can add basic/advanced create at any time, right? They don't have to be completely different experiences to start with - you can slowly build up the defaults for 'basic' and the extra prompts for 'advanced'. To start with, the only difference might be these three settings and the auto-detected remote logic